src/hooks/useTxNotifications.ts

```diff
@@ -25,21 +27,32 @@ enum Variant {
 
 const useTxNotifications = (): void => {
   const dispatch = useAppDispatch()
+  const safeAddress = useSafeAddress()
```

Comment:
We should use `const { safeAddress } = useSafeInfo()` since `useSafeAddress` relies on the URL which caused some issues with hydration if I remember correctly.

Reply:
It’s fine to use the url one.